lib/solar_system.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'planet'
+
+class SolarSystem
+  attr_reader :star_name, :planets
+  attr_writer :planets
```

Comment:
You shouldn't need a writer for `planets`

Reply:
Yes, correct, forgot to delete after tests.